app/javascript/components/close_contact/CloseContact.js

```diff
@@ -166,12 +253,7 @@ class CloseContact extends React.Component {
           <Button variant="secondary btn-square" onClick={toggle}>
             Cancel
           </Button>
-          <Button
-            variant="primary btn-square"
-            onClick={submit}
-            disabled={
-              !(this.state.first_name || this.state.last_name || this.state.email || this.state.primary_telephone || this.state.notes) || this.state.loading
-            }>
+          <Button variant="primary btn-square" onClick={submit} disabled={!(this.state.first_name && this.state.last_name) || this.state.loading}>
```

Comment:
It comes from `SARAALERT_1099`. Kelly said the required fields should be `first name, last name, email or phone number.`. The OR clause makes the logic tricky, but since I immediately start `1099` after `616` I combined the easy part into `616` to get that in early. Also since i was updating the required fields from this ticket anyway, i figured I might as well make it consistent. 

If you'd like I can revert these changes so it's back to the old incorrect validation. And then apply them to my other branch with all the complex validation logic.

Reply:
1099 says:
> Require one of the following fields to have an input before it can be saved: first name, last name, email or phone number.
I wouldn't update any of this here, it's a separate ticket and as it is now is incorrect for 1099 anyway.

Reply:
Roger that! Reverting to the older logic

Reply:
You'll need to update the schema as well, right now it still marks first and last as both required.

Reply:
`Master` does have them as required fields, but sounds good! Removing any `required` attributes from the close contact yup schema